### ISSUE TITLE
Several controls don't meet new non-text contrast ratio requirement

### ISSUE THREAD

**Issue by YuliKl** [microsoft/microsoft-ui-xaml#1233]
**Describe the bug**

Controls will be required to meet [MAS 1.4.11 - Non-text Contrast](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B6CA3BF14-2635-434A-832C-28DD9A18B9FC%7D&file=MAS%201.4.11%20%E2%80%93%20Non-text%20Contrast.docx&action=default&mobileredirect=true&cid=e2ed3f7b-f231-4f75-a423-80c7558563da)

### Slider
In light theme, the Slider control's grey track presents a contrast ratio of 2.8:1 instead of the required 3:1.

![image](https://user-images.githubusercontent.com/13142665/63885341-0308f080-c98d-11e9-8b9c-bc87b0824dc3.png)

### Buttons
Button backplates do not meet contrast ratios in either light or dark theme.
![image](https://user-images.githubusercontent.com/13142665/63890674-559bda00-c998-11e9-8a99-2e2417e23e56.png)

**Comment by mdtauk** [comment]
Perhaps this is something that could be addressed through Issues #920 and #921 where I suggest another look at Light and Dark Theme colours

**Comment by jevansaks** [comment]
@yulikl do you have a suggestion for how to fix? Are there comps with the updates?

**Comment by mdtauk** [comment]
Any contrast ratio check will differ if the control is not placed on a Pure White (or Pure Black in dark mode) background.

The Outline style would probably fair better for contrast, and using Semibold text for the button text, would also help pass the ratio test at 14 epx and above.

**Comment by chingucoding** [comment]
@YuliKl How could this be fixed? Should we pick a darker gray for the track so we meet the new requirement?

**Comment by YuliKl** [comment]
Closing this issue. No changes to Button colors are required as clarified in the WCAG 2.1 guidance. I'm not sure that anyone uses sliders with tick marks as I have seen no reports of violations there.
